js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -0,0 +1,131 @@
+        <a href='/options' class='active'>Options</a>
+    </nav>
+
+    <main id='ContentArea'>
+        <h1>You can disable Immersive Reader features as desired!</h1>
+        <p>These are some of the features you can disable:</p>
+        <p><code>disableGrammar</code> Parts of speech, syllables and Picture Dictionary are not going to be available.</p>
+        <p><code>disableLanguageDetection</code> Prevents Immersive Reader to detect the content language.</p>
+        <p><code>disableTranslation</code> Translation controls are going to be disabled.</p>
+        <h3>Examples:</h3><br />
+        <section class='section' id='Sample1'>
+            <div class='immersive-reader-button ir-button-small' onclick='handleLaunchImmersiveReader(1)'></div>
+            Specify your content's speech language: <input id='lang1' value='en' /> (i.e 'fr', 'es', 'de', 'zh')
+            <h3 id='title1'>Geography (Content must be read in the specified language)</h3>
+            <p id='text1'>The study of Earth's landforms is called physical geography. Landforms can be mountains and valleys. They can also be glaciers, lakes or rivers. Landforms are sometimes called physical features. It is important for students to know about the physical geography of Earth. The seasons, the atmosphere and all the natural processes of Earth affect where people are able to live. Geography is one of a combination of factors that people use to decide where they want to live.</p>
+        </section>
+        <div id='Cols'>
+            <section class='section' id='Sample2'>
+                <div class='immersive-reader-button ir-button-small' onclick='handleLaunchImmersiveReader(2)'></div>
+                <h3 id='title2'>Water (open UI without Grammar, syllables and Picture Dictionary features - <span id='lang2'>en</span>)</h3>
+                <p id='text2'>Fresh water sources also influence where people settle. People need water to drink. They also need it for washing. Throughout history, people have settled near fresh water. Living near a water source helps ensure that people have the water they need.</p>
+            </section>
+            <section class='section' id='Sample3'>
+                <div class='immersive-reader-button ir-button-small' onclick='handleLaunchImmersiveReader(3)'></div>
+                <h3 id='title3'>Mountain Ranges (open UI without Translation feature- <span id='lang3'>en</span>)</h3>
+                <p id='text3'>The physical features of a region are often rich in resources. Within a nation, mountain ranges become natural borders for settlement areas. In the U.S., major mountain ranges are the Sierra Nevada, the Rocky Mountains, and the Appalachians.</p>
+            </section>
+        </div>
+    </main>
+
+    <script type='text/javascript'>
+        async function handleLaunchImmersiveReader(sampleId) {
+            var langEl = $('#lang' + sampleId);
+            var lang = langEl.val() || langEl.text();
```

Comment:
why check both, val() || text?

Reply:
I think because some lang elements are span, and other is input, trying to keep just one type...

Reply:
OK, pls add a comment beside this line

Reply:
need to keep both because for disableLanguage detection the lang is specified by the user in an "input".
for the other samples is fixed to english and specified in "span" elements.